docs/src/main/sphinx/sql/explain-analyze.rst

```diff
@@ -83,20 +99,58 @@ For example, the window function operator will output the following::
 
 .. code-block:: text
 
-                                              Query Plan
-    -----------------------------------------------------------------------------------------------
-      ...
-             - Window[] => [clerk:varchar(15), count:bigint]
-                     Cost: {rows: ?, bytes: ?}
-                     CPU fraction: 75.93%, Output: 8130 rows (230.24kB)
-                     Input avg.: 8130.00 lines, Input std.dev.: 0.00%
-                     Active Drivers: [ 1 / 1 ]
-                     Index size: std.dev.: 0.00 bytes , 0.00 rows
-                     Index count per driver: std.dev.: 0.00
-                     Rows per driver: std.dev.: 0.00
-                     Size of partition: std.dev.: 0.00
-                     count := count("clerk")
-     ...
+                                                                                                      Query Plan
+    ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+     Fragment 1 [SINGLE]
+         CPU: 55.78ms, Scheduled: 112.49ms, Input: 8130 rows (158.79kB); per task: avg.: 8130.00 std.dev.: 0.00, Output: 8130 rows (71.46kB)
+         Output layout: [count]
+         Output partitioning: SINGLE []
+         Stage Execution Strategy: UNGROUPED_EXECUTION
+         Project[]
+         │   Layout: [count:bigint]
+         │   Estimates: {rows: ? (?), cpu: ?, memory: ?, network: ?}
+         │   CPU: 5.00ms (0.45%), Scheduled: 22.00ms (0.45%), Output: 8130 rows (71.46kB)
+         │   Input avg.: 508.13 rows, Input std.dev.: 387.30%
+         └─ LocalExchange[ROUND_ROBIN] ()
+            │   Layout: [clerk:varchar(15), count:bigint]
+            │   Estimates: {rows: ? (?), cpu: ?, memory: ?, network: ?}
+            │   CPU: 0.00ns (0.00%), Scheduled: 0.00ns (0.00%), Output: 8130 rows (230.24kB)
+            │   Input avg.: 8130.00 rows, Input std.dev.: 0.00%
+            └─ Window[]
+               │   Layout: [clerk:varchar(15), count:bigint]
+               │   CPU: 38.00ms (3.43%), Scheduled: 54.00ms (1.09%), Output: 8130 rows (230.24kB)
+               │   Input avg.: 8130.00 rows, Input std.dev.: 0.00%
+               │   Active Drivers: [ 1 / 1 ]
+               │   Index size: std.dev.: 0.00 bytes, 0.00 rows
+               │   Index count per driver: std.dev.: 0.00
+               │   Rows per driver: std.dev.: 0.00
+               │   Size of partition: std.dev.: 0.00
+               │   count := count("clerk") RANGE UNBOUNDED_PRECEDING CURRENT_ROW
```

Comment:
Leave only L119-L128 as the previous plan because we want to show `Window[]` example here.

Reply:
> ... tpch:orders:sf0.01 ...

The above line implies you used `tiny` schema. Please use `sf1` and show only `Window[]` (L119-L128 ).